lib/customer.rb

```diff
@@ -0,0 +1,34 @@
+require_relative 'order'
+require_relative 'menu'
+require_relative 'confirmation_text_sender'
+require 'csv'
+
+class Customer
+
+  attr_reader :menu
+  attr_accessor :order_created, :pending_order, :selection
+
+  def initialize()
+    @pending_order = []
+    @menu = Menu.new
+    @order_created = nil
+    @selection = selection
+  end
+
+  def select_dish
```

Comment:
The `Customer` class should not be responsible for the user interface.  The UI can be added as a layer later on.